spec/acceptance/site_layout_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'landing page', type: :feature do
+  before do
+    visit root_path
+  end
+  context 'When user not logged in' do
+    it { expect(page).to have_content 'Sign up' }
+    it { expect(page).to have_content 'login' }
+    it { expect(page).to have_css '.image' }
+  end
+
+  context 'When user is logged in' do
```

Comment:
You have 2 contexts, but they are the same. Is a good practice, right under the context line, to set the conditions that determine the context.